src/main/java/hudson/plugins/doxygen/DoxygenBuilder.java

```diff
@@ -95,70 +97,62 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         try {
             if (!doxygenPath.exists()) {
                 listener.fatalError(String.format("The path to Doxygen executable doesn't exist : \"%s\"", doxygenPath));
-                return false;
             }
         } catch (IOException e) {
             listener.fatalError(String.format("Failed checking the existence of Doxygen executable : \"%s\"", doxygenPath));
             Util.displayIOException(e, listener);
-            return false;
+            return;
         }
-        
+
         // Raise an error if the doxyfile isn't set
         String pathToDoxyfile = getDoxyfilePath();
         if (pathToDoxyfile == null || pathToDoxyfile.trim().length() == 0)
         {
             listener.fatalError("The Doxyfile path is empty. Please review the build step configuration.");
         }
-        
+
         // Raise an error if the doxyfile path doesn't exists
         FilePath doxyfile = new FilePath(launcher.getChannel(), pathToDoxygen);
         try {
             if (!doxyfile.exists()) {
                 listener.fatalError(String.format("The path to Doxyfile doesn't exist : \"%s\"", doxyfile));
-                return false;
+                return;
             }
         } catch (IOException e) {
             listener.fatalError(String.format("Failed checking the existence of the Doxyfile : \"%s\"", doxyfile));
             Util.displayIOException(e, listener);
-            return false;
+            return;
         }
-        
+
         // Build the command line to run
         ArgumentListBuilder args = new ArgumentListBuilder();
         args.add(pathToDoxygen);
         args.add(pathToDoxyfile);
-        FilePath executionDirectory = build.getWorkspace();
+        FilePath executionDirectory = workspace;
 
         try {
             listener.getLogger().println(String.format("Executing the command %s from %s", args.toStringWithQuote(), executionDirectory));
             // Doxygen output parser to handle warnings and errors
-            DoxygenConsoleParser outputParser = new DoxygenConsoleParser(listener.getLogger(), build.getCharset());
+            DoxygenConsoleParser outputParser = new DoxygenConsoleParser(listener.getLogger(), run.getCharset());

```

Comment:
This is probably wrong. You should rather look up the charset expected on that computer.